src/test/java/uk/gov/companieshouse/itemhandler/kafka/OrdersKafkaConsumerTest.java

```diff
@@ -0,0 +1,118 @@
+package uk.gov.companieshouse.itemhandler.kafka;
+
+import org.hamcrest.Matchers;
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.kafka.consumer.resilience.CHKafkaResilientConsumerGroup;
+import uk.gov.companieshouse.kafka.exceptions.SerializationException;
+import uk.gov.companieshouse.kafka.message.Message;
+import uk.gov.companieshouse.kafka.serialization.AvroSerializer;
+import uk.gov.companieshouse.kafka.serialization.SerializerFactory;
+import uk.gov.companieshouse.orders.OrderReceived;
+
+import java.util.concurrent.ExecutionException;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class OrdersKafkaConsumerTest {
+    private static final String ORDER_RECEIVED_URI = "/order/ORDER-12345";
+    private static final String ORDER_RECEIVED_TOPIC = "order-received";
+    private static final String ORDER_RECEIVED_TOPIC_RETRY = "order-received-retry";
+    private static final String ORDER_RECEIVED_TOPIC_ERROR = "order-received-error";
+
+    @Spy
+    @InjectMocks
+    private OrdersKafkaConsumer ordersKafkaConsumer;
+    @Mock
+    private CHKafkaResilientConsumerGroup chKafkaConsumerGroupMain;
+    @Mock
+    private CHKafkaResilientConsumerGroup chKafkaConsumerGroupRetry;
+    @Mock
+    private SerializerFactory serializerFactory;
+    @Mock
+    private AvroSerializer serializer;
+    private static final String expectedMessageValue = "$/order/ORDER-12345";
```

Comment:
Why does this differ slightly from `ORDER_RECEIVED_URI`?

Reply:
It is how the serialized value looks like post serialization by `AvroSerializer`.

Reply:
* To a code reader it is slightly concerning that "what comes out" does not appear to match "what went in". Is it possible to deserialise it or something so that the values are the same? (If this is not too much of a make-work).
* To change the subject slightly, if we leave this value in as-is then in Java it should probably be called `EXPECTED_MESSAGE_VALUE`.

Reply:
That makes sense. I have updated the code.